app/qml/map/MMPositionMarker.qml

```diff
@@ -8,35 +8,32 @@
  ***************************************************************************/
 
 import QtQuick
-import lc 1.0
 
 Item {
-  id: control
+  id: root
 
   required property real xPos
   required property real yPos
   property real direction: 0
   property int size: 24 * __dp
-  property real screenAccuracy: 50
+  property real accuracyRingSize: 50
 
   property bool trackingMode: false
   property bool withAccuracy: true
   property bool hasPosition: true
   property bool hasDirection: true
   property real horizontalAccuracy: 0.5
 
+  visible: root.hasPosition
+
   Rectangle {
     id: accuracyIndicator
 
-    x: control.xPos - width / 2
-    y: control.yPos - height / 2
-    width: control.screenAccuracy
+    x: root.xPos - width / 2
+    y: root.yPos - height / 2
+    width: root.accuracyRingSize
     height: width
-    visible: withAccuracy &&
-             control.hasPosition &&
-             (control.horizontalAccuracy > 0) &&
-             (accuracyIndicator.width > control.size / 2.0)
```

Comment:
Why did we remove all of these lines? I though we would remove just the `root.hasPosition` part.